src/integrationTest/sam.test.ts

```diff
@@ -62,6 +65,7 @@ const scenarios: TestScenario[] = [
         debugSessionType: 'pwa-node',
         language: 'javascript',
         dependencyManager: 'npm',
+        vscodeMinimum: '1.50',
```

Comment:
```suggestion
        vscodeMinimum: '1.50.0,
```

`semver` expects all parts of a version to be present it seems.